### ISSUE TITLE
Unable to execute

### ISSUE THREAD

**Issue by montanaflynn** [vektorlab/slackcat#14]
Hi I followed the README but hit this error. MacBook running El Capitan.

```
curl -sLo slackcat https://github.com/vektorlab/slackcat/releases/download/v0.7/slackcat-0.7-darwin-amd64
sudo mv slackcat /usr/local/bin/
sudo chmod +x /usr/local/bin/slackcat

> slackcat
Failed to execute process '/usr/local/bin/slackcat'. Reason:
exec: Exec format error
The file '/usr/local/bin/slackcat' is marked as an executable but could not be run by the operating system.

**Comment by montanaflynn** [comment]
So I think the curl command failed and didn't show an error because I got it with wget and it worked fine.

**Comment by bcicen** [comment]
Resolved in #15
